### PR TITLE
Feature/moca 1546

### PR DESCRIPTION
**Pull request by reinieravila**

### CODE REVIEW

File: oauth2.md [hunk 1]
```diff
@@ -114,19 +114,26 @@ POST https://oauth.mocaverse.xyz/api/oauth/token
...
-POST https://oauth.mocaverse.xyz/api/oauth/token?grant_type=authorization_code&client_id=EXAMPLE_CLIENT_ID&client_secret=MOCA_SECRET&code=602cbbe1-60dd-482f-b0d8-329f5f1254c7&redirect_uri=https%3A%2F%2Fyour-app.example.com%2Fcallback
+POST https://oauth.mocaverse.xyz/api/oauth/token
+```
+
+**Request Body**
+
+```
+grant_type=authorization_code&client_id=EXAMPLE_CLIENT_ID&client_secret=MOCA_SECRET&code=602cbbe1-60dd-482f-b0d8-329f5f1254c7&redirect_uri=https%3A%2F%2Fyour-app.example.com%2Fcallback
 ```
+
```
Comment (reinieravila) [inline oauth2.md:132]: Relevant change. Others are automatically formatted tables.

Comment (abcalebh) [reply oauth2.md:132]: also add header in this example

Comment (reinieravila) [reply oauth2.md:132]: Updated

File: oauth2.md [hunk 2]
```diff
@@ -112,21 +105,31 @@ https://your-app.example.com/callback?error=access_denied&state=E0BXkRkKnvqMiDdY
...
-Content-type: `application/x-www-form-urlencoded`
 
-| Parameter       | Description                                                                            |
-|-----------------|----------------------------------------------------------------------------------------|
-| grant_type    | must be `authorization_code`                                                           |
-| client_id     | your `client_id` as provided during [Set up](#set-up)             |
-| client_secret | your `client_secret` as provided during [Set up](#set-up)          |
+| Header       | Description                         |
+|--------------|-------------------------------------|
+| Content-Type | `application/x-www-form-urlencoded` |
```
Comment (reinieravila) [inline oauth2.md:111]: the added header info

**Review by m1ngshum — APPROVED** [review]

**Pull request merged**